core/server/config/index.js

```diff
@@ -1,11 +1,13 @@
 var Nconf = require('nconf'),
     path = require('path'),
-    debug = require('debug')('ghost:config'),
+    _debug = require('debug'),
```

Comment:
yes true, `debug.enabled` is not available as it operates on the global package and not on the debug instance

> Also we should definitely do that, if we want to, separately :)

👍